courses/free-courses-ml.md

```diff
@@ -54,15 +72,29 @@
 * [Data Structure Challenge](https://youtube.com/playlist?list=PLY-ecO2csVHc5kajCd3fHU6MhkTABkRh9) - Brototype Malayalam
 
 
+### Docker
+
+* [Docker Tutorial](https://www.youtube.com/playlist?list=PLmFap_OoBJh7Z5tTSkeXciuIMZnKdJ1uu) - Tech Trivandrum
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.